src/overlays/actors/ovl_En_Slime/z_en_slime.c

```diff
@@ -10,30 +10,62 @@
 
 #define THIS ((EnSlime*)thisx)
 
+#define ICE_BLOCK_UNUSED 255
+#define ICE_BLOCK_TIMER_MAX 254
```

Comment:
This is great, but I have one suggestion:
```suggestion
#define ICE_BLOCK_TIMER_MAX 254
#define ICE_BLOCK_UNUSED (ICE_BLOCK_TIMER_MAX + 1)
```

This way, in a modding context, someone can increase `ICE_BLOCK_TIMER_MAX` if they want, and then `ICE_BLOCK_UNUSED` will still work properly, no matter what they end up increasing it to.

Reply:
Ah good idea. I can do that.
One thing to note though is that the struct variable iceBlockTimer is a u8, so a modder would have to remember to increase the size of that if they wanted to increase the timer beyond 254.

Reply:
Ah, fair point. Still, if they want to bump it up to an `s16`, for example, this will still come in handy.